### PR TITLE
Fix manual init with auto performance for RN and iOS

### PR DESCRIPTION
**Pull request by marandaneto**
https://github.com/getsentry/sentry-react-native/pull/2517
Reason https://github.com/getsentry/sentry-cocoa/issues/2262

### CODE REVIEW

**Comment by krystofwoldrich** [comment]
There are is nothing required on the Android side when initialized manually?

File: src/platforms/react-native/manual-setup/native-init.mdx [hunk 1]
```diff
@@ -25,6 +25,11 @@ Do not use `autoInitializeNativeSdk` to disable the native layer, instead use [e
 
 Next, initialize the native SDKs for Android, which is documented below, or for iOS by following the [configuration guide to initialize the Sentry Cocoa SDK](/platforms/apple/guides/ios/#configure). Note that you do not need to install the native SDKs as they are already packaged with the React Native SDK.
 
+## iOS Configuration
+
+To enable the `App Start Instrumentation` and `Slow and Frozen Frames` on iOS when initing the Sentry Cocoa SDK manually,
+you have to [configure the Sample Rate](/platforms/apple/performance/#configure-the-sample-rate/).
```
Comment (imatwawana) [inline src/platforms/react-native/manual-setup/native-init.mdx:31]:
```suggestion
you have to [configure the sample rate](/platforms/apple/performance/#configure-the-sample-rate/).
```

**Review by imatwawana — APPROVED** [review]
Just some style and wording edits

**Comment by marandaneto** [comment]
> There are is nothing required on the Android side when initialized manually?

nope

**Pull request merged**